core/domain/topic_domain.py

```diff
@@ -565,27 +565,6 @@ def require_valid_name(cls, name):
                 'Topic name should be at most %d characters, received %s.'
                 % (name_limit, name))
 
-    @classmethod
-    def require_valid_abbreviated_name(cls, name):
```

Comment:
While I agree with this deletion in general, why is it part of this PR? /cc @aks681

Reply:
Yeah, there are instances of abbreviated_names in some locations still (#9102 is tracking its removal). It's fine to remove this function as it  isn't used anywhere.